scripts/dap-ami-bootstrap.sh

```diff
@@ -11,27 +11,36 @@ if ! docker images | grep appliance; then
 fi
 
 image_id=$(docker images -q)
-container_name='dap-appliance'
+container_name='dap'
 
 docker rm -f ${container_name} > /dev/null 2>&1 || true
 
 mkdir -p /var/log/conjur
 mkdir -p /opt/conjur/backup
+mkdir -p /opt/cyberark/dap/configuration
+mkdir -p /opt/cyberark/dap/security
+mkdir -p /opt/cyberark/dap/backup
+mkdir -p /opt/cyberark/dap/seeds
+mkdir -p /opt/cyberark/dap/logs
 
 echo "Creating DAP container"
 cid=$(docker create \
 --name ${container_name} \
---privileged --restart always \
+--privileged --restart unless-stopped \
 --log-driver=journald \
--v /var/log/conjur:/var/log/conjur \
--v /opt/conjur/backup:/opt/conjur/backup \
--p "443:443" \
--p "444:444" \
--p "636:636" \
--p "5432:5432" \
--p "5433:5433" \
--p "1999:1999" \
--p "127.0.0.1:38053:38053" \
+--volume /var/log/conjur:/var/log/conjur:Z \
+--volume /opt/conjur/backup:/opt/conjur/backup:Z \
+--volume /opt/cyberark/dap/configuration:/opt/cyberark/dap/configuration:Z \
+--volume /opt/cyberark/dap/security:/opt/cyberark/dap/security:Z \
+--volume /opt/cyberark/dap/backup:/opt/cyberark/dap/backup:Z \
+--volume /opt/cyberark/dap/seeds:/opt/cyberark/dap/seeds:Z \
+--volume /opt/cyberark/dap/logs:/opt/cyberark/dap/logs:Z \
```

Comment:
What is the value of these if `/opt/cyberark/dap` doesn't exist in the appliance image, particularly in regards to `/log` and `/backup`?

The appliance isn't going to write to these directories, without some other manipulation of the appliance itself. I'm worried seeing these in the docker run command will be misleading.

Reply:
We've gotta start somewhere, why not here? And without the security one we have no place for the seccomp, so might as well include the rest.

Reply:
It's really just `/logs` and `/backup` I'm concerned about, because they're for files coming from the container to the host. `/opt/conjur/backup` and `/var/log/conjur` are significant paths within the container itself that it will write to. So the `/opt/cyberark/dap` variants become essentially no-ops.

(Side note, if kept, `/opt/cyberark/dap/logs` should really probably be `/var/log/cyberark/dap`)

I don't really have an opinion on the others, I agree they should exist as ways to get files from the host to the container. And these suggestions for that purpose are A+ 😄.